client/src/partials/cash/extra_payment/extra_payment.html

```diff
@@ -1,163 +1,212 @@
+                  </div>
+                </div>
+              </div>
 
-  <div class="pull-right">
-    <button
-      style="margin-left: 5px;"
-      ng-click="reconfigure()"
-      class="btn btn-sm btn-default">
-      {{'PRIMARY_CASH.EXPENSE.SELECT_ACCOUNT' | translate}}
-      <span class="glyphicon glyphicon-repeat"></span>
-    </button>
-  </div>
-</nav>
-
-<main>
-  <div ng-if="!session.configured" class="row margin-top-10">
-    <div class="col-xs-12">
-      <div class="panel panel-default">
-        <div class="panel-heading">
-          {{ "PRIMARY_CASH.EXPENSE.SELECT_ACCOUNT" | translate }}:
-          <b>{{ "CASH.EXTRA.WAIT_ACCOUNT" | translate }}</b>
-        </div>
+              <div class="form-group">
+                <label class="control-label col-xs-4 required">
+                  {{ "SELECT.ACCOUNT" | translate }}
+                </label>
+                <div>
+                  <div class="pull-left form-group input-group">
+                  <span class="input-group-addon">
+                    <span class="glyphicon glyphicon-briefcase"></span>
+                  </span>  
+                  <input class="form-bhima" ng-model="session.account" required typeahead="account as formatAccount(account) for account in session.accounts.data | filter:$viewValue | limitTo:10" typeahead-template-url="templateAccount.html" placeholder="{{ 'PATIENT_STANDING.PLACEHOLDER' | translate }}">
+                  </div>
+                </div>
+              </div>
+              <div class="pull-right">
+                <button class="btn btn-sm btn-success" type="button" ng-click="search()" ng-disabled="configForm.$invalid">
+                    {{ "REPORT.GENERATE" | translate }}
+                </button>
+              </div>
 
-        <div class="panel-body">
-          <div class="configure">
-            <span
-              style="font-size: 70px;"
-              class="glyphicon glyphicon-inbox">
-            </span>
-          </div>
-          <br>
-          <div class="input-group">
-            <input
-              class="form-bhima"
-              type="text"
-              ng-model="session.ac"
-              placeholder="Selectionner compte"
-              typeahead="ac as formatAccount(ac) for ac in session.accounts | filter:$viewValue | limitTo:10" typeahead-template-url="AccountList.html"
-              typeahead-on-select="setConfiguration(session.ac)"
-              >
-            <span class="input-group-btn">
-              <button ng-click="setConfiguration(session.ac)" class="btn btn-default btn-sm square">{{ "FORM.SUBMIT" | translate }}</button>
-            </span>
+            </form>
           </div>
         </div>
       </div>
-    </div>
+      <script type="text/ng-template" id="template.html">
```

Comment:
It would be good to make this template an html template somewhere.  What typeahead uses it?